fe/fe-core/src/main/java/org/apache/doris/statistics/ColumnStatistic.java

```diff
@@ -154,7 +154,8 @@ public static ColumnStatistic fromResultRow(ResultRow resultRow) {
             }
             columnStatisticBuilder.setSelectivity(1.0);
             columnStatisticBuilder.setOriginalNdv(ndv);
-            Histogram histogram = Env.getCurrentEnv().getStatisticsCache().getHistogram(tblId, idxId, colName);
+            Histogram histogram = Env.getCurrentEnv().getStatisticsCache().getHistogram(tblId, idxId, colName)
```

Comment:
maybe the better way is, one class use for caching column stats, one class use for caching histogram, and a new class use for stats derive, it contains both column stats and histogram

Reply:
There is not much difference between these two implementations. I might make the changes as you suggested when it becomes necessary to do so.